.circleci/config.yml

```diff
@@ -6,8 +6,8 @@ executors:
       - image: docker.mirror.hashicorp.services/circleci/golang:1.14
     environment:
       TEST_RESULTS: /tmp/test-results # path to where test results are saved
-      CONSUL_VERSION: 1.9.0-rc1 # Consul's OSS version to use in tests
-      CONSUL_ENT_VERSION: 1.9.0+ent-rc1 # Consul's enterprise version to use in tests
+      CONSUL_VERSION: 1.9.4 # Consul's OSS version to use in tests
```

Comment:
So now I'm wondering if setting tproxy to default enabled will work without pointing this at the beta build of consul 1.10? @ishustava ?

Reply:
it should. The api changes are backward compatible and so the newer API package should work against the older agent.

Reply:
perfect, okay thanks for clarifying!